docs/managing-e2e-tests.md

```diff
@@ -0,0 +1,166 @@
+
+Tests can exist in multiple dashboards at once. For example a test defined like:
+```
+- name: kubeadm-gce-1.12
+  test_group_name: ci-kubernetes-e2e-kubeadm-gce-1-12
+```
+can exists in both the `sig-cluster-lifecycle-all` and `sig-release-master-blocking` dashboards. Essentially both tests would show details about the same test job called `ci-kubernetes-e2e-kubeadm-gce-1-12`.
+
+### Updates to kubeadm tests
+
+Before each new Kubernetes release and during the second month of the [release-cycle](https://github.com/kubernetes/kubeadm/blob/master/docs/release-cycle.md), a set of manual actions have to be performed, so that the kubeadm e2e tests are up to date with the new release.
+
+The operation can be broken down in 3 steps:
+1) Updating the kubeadm test jobs and parameters.
+2) Updating testgrid to reflect the changes.
+3) Finalizing the changes.
+
+#### Updating test jobs
+
+This document will cover information on how to perform updates on these 3 files:
+- [kubeadm.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm.yaml "kubeadm.yaml")
+
+Holds test jobs where the kubeadm version matches the Kubernetes control-plane and the kubelet versions.
+
+- [kubeadm-upgrade.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-upgrade.yaml "kubeadm-upgrade.yaml")
+
+Holds test jobs that perform a upgrade from version X to version Y (usually `Y = X + 1`).
+
+- [kubeadm-x-on-y.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-x-on-y.yaml "kubeadm-x-on-y.yaml")
+
+Holds test jobs that run kubeadm version Y, on a control plane and kubelet version X,
+as kubeadm does support `Y = X + 1`
+
+The files can be found in the [sig-cluster-lifecycle](https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes/sig-cluster-lifecycle) folder.
+
+The idea is to remove old test jobs and add new ones. If Kubernetes 1.15 is about to be released soon, this means that `15 - 3 = 12` MINOR versions would go outside of the support skew of the project (the project would support `15 - 2 = 13` So, `1.12` jobs have to be removed and `1.15` jobs have to be added.
+
+- If there are `1.12 -> 1.13` upgrade jobs or `1.12 on 1.13` jobs leave them, because they still contain a supported version `1.13`.
+- Remove plain `1.12` jobs.
+- Add upgrade jobs for `1.14 -> 1.15` add `1.14 on 1.15` jobs.
+- Add plain `1.15` jobs.
+
+Each job contains some versioned labels and branches passed as properties, that are not going to be outlined in this document, but they need updates too:
+- `release-1.14`
+- `release/latest-1.14`
+- `ci/latest-bazel-1.14`
+- `latest-bazel-1.14`
+- ...
+
```

Comment:
@fabriziopandini hm, i have no information for this new command. is there are KEP or a tracking issue?

Reply:
@fabriziopandini 
tempted to not cover the `kubeadm-network` file since @timothysc  mentioned that those should be sig-network responsibility. yet, the file still resides in the [sig-cluster-lifecycle folder](https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes/sig-cluster-lifecycle) and as i mentioned on slack those tests use k-a.

Reply:
moving the file to sig-network's folder.